qemu/tests/qsd_hotplug_vubp.py

```diff
@@ -0,0 +1,116 @@
+"""QSD hotplug vhost-user-blk-pci device test"""
+import time
+
+from provider.qsd import QsdDaemonDev, add_vubp_into_boot
+from provider.qsd import plug_vubp_devices, unplug_vubp_devices
+from virttest import error_context, env_process, utils_disk, utils_misc
+from virttest.utils_disk import clean_partition_windows
+
+
+@error_context.context_aware
```

Comment:
no error_context.context() statement in codes, I think we can remove it. Others LGTM.

 (1/2) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qsd_hotplug_vubp.q35: PASS (145.65 s)
(2/2) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.qsd_hotplug_vubp.q35: PASS (284.71 s)

Reply:
updated